docs/architecture/specs/Spec-DID.md

```diff
@@ -116,9 +116,106 @@ A DDO document is composed of standard DDO attributes:
 * `proof`
 * `verifiableCredential`
 * `service`
+* `_nvm`
 
 Asset metadata can be included as one of the objects inside the `"service"` array, with type `"metadata"`.
 
+
+#### Nevermined internal attributes
+
+Nevermined registers some internal attributes in the asset metadata. They allow to store information that makes easier to understand the meaning of an asset, their life-cycle and where they can be used. The internal attributes are recorded under the `_nvm` section located at the root of the DDO.
```

Comment:
I guess the _nvm section is internal to nevermined and cannot be changed by the user.

Reply:
Yep, that's the intention. Added in any case a clarification about that